test/index.js

```diff
@@ -1,3 +1,5 @@
+import 'core-js/stable'
```

Comment:
👍 this is porbably a good idea 😅

Reply:
I can `t get it. Should i install corejs or add in babel preset?

Reply:
Importing core-js in the test suite I think is fine.